CHANGELOG.md

```diff
@@ -1,5 +1,107 @@
 # Changelog
 
+## ☀️ 0.5.0
+
+- ### ✨ Features
+
+  - #### 🍱 Module Support
+
+    - **BREAKING: use correct `package.json` keys for generated JavaScript - [ashleygwilliams], [issue/309] [pull/312]**
+
+      [issue/309]: https://github.com/rustwasm/wasm-pack/issues/309
+      [pull/312]: https://github.com/rustwasm/wasm-pack/pull/312
+
+  - #### 🛠️ New Commands
+
+    - **`wasm-pack init` is now `wasm-pack build` - [csmoe], [issue/188] [pull/216]**
+
+      [csmoe]: https://github.com/csmoe
+      [issue/188]: https://github.com/rustwasm/wasm-pack/issues/188
+      [pull/216]: https://github.com/rustwasm/wasm-pack/pull/216
+
+    - **add new command: `wasm-pack test` - [fitzgen], [pull/271]**
+
+      [pull/271]: https://github.com/rustwasm/wasm-pack/pull/271
+
+    - **add OTP support to `wasm-pack publish` - [mstallmo], [issue/257] [pull/282]**
```

Comment:
I think it is worth expanding "OTP", since that acronym migth not be known to everyone (I think "open telecom platform" and erlang, for example).

Reply:
I think One True Pairing because of my Tumblr years so expanding then putting (OTP) would help